pkg/fleet/installer/service/embedded/datadog-installer-exp.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Datadog Installer Experiment
+After=network.target
+OnFailure=datadog-installer.service
+Conflicts=datadog-installer.service
+JobTimeoutSec=3000 #50 minutes
+
+[Service]
+Type=oneshot
+PIDFile=/var/run/datadog/installer/installer.pid
+ExecStart=/opt/datadog-packages/datadog-installer/stable/bin/installer/installer run -p /var/run/datadog/installer/installer.pid
+ExecStart=/opt/datadog-packages/datadog-installer/stable/bin/installer/installer run -p /var/run/datadog/installer/installer.pid
+ExecStart=/opt/datadog-packages/datadog-installer/stable/bin/installer/installer run -p /var/run/datadog/installer/installer.pid
```

Comment:
Should't the path be `/opt/..../experiment/bin/...` here?

Reply:
good catch